### ISSUE TITLE
ng-model hard-coded string

### ISSUE THREAD

**Issue by badmacstone** [darylrowland/angucomplete#92]
I was wondering how I could set the ng-model of the input-field at my own. I thought the selectedObject would be do that thing but then I saw that the ng-model was hard coded ("searchString"). Is there a possibility to change that or am I doing something wrong?

**Comment by mgardner-git** [comment]
Yes, I think I have the same problem.  The ang-complete field does not update correctly if you have an ng-model attribute set on it.
